### DISCUSSION TITLE
Instalation trouble

### DISCUSSION THREAD

**Discussion by Ewwe** [MrFibunacci/laravel-form-components#18] [Q&A] [answered]
Hi, i really wanna use your library. But after going through installation im getting spinner with "Loading in progress...".
Have i done something wrong ? I never use livewire in laravel so idk.

Im using dev version 5 bcs laravel 9. But same trouble ive got in lover versions... 

I found that property $initialized of Okipa\LaravelTable\Livewire\Table is problem. 

Thanks :)

**Comment by Okipa** [comment, marked as answer]
Hi @Ewwe, this discussion has been opened on the wrong package because of a wrong issue-template on my `laravel-table` package, sorry for that.

I close it but you should know that v5 of laravel-table is not released as stable yet so you shouldn't use it for another purpose than testing the package.

You also should know that you have to follow the documentation of Livewire to install this dependency before trying to use v5 of laravel table. It will not be functional if you do not install it properly (try to implement the Livewire example component, if it works, laravel table v5 will also work).
